Cargo.toml

```diff
@@ -19,13 +19,12 @@ codegen = ["swc_ecma_codegen", "swc_ecma_codegen_macros", "swc_macros_common"]
 compat = ["transforms", "swc_ecma_transforms_compat", "swc_trace_macro", "swc_config", "swc_config_macro"]
 dep_analysis = ["visit"]
 dep_graph = ["swc_ecma_dep_graph"]
-module_specifier = ["deno_media_type/module_specifier", "url"]
```

Comment:
Diagnostics require a URL and I think it's time to just always include this and make it mandatory to just use ModuleSpecifier. We do it anyway in all the crates.